57606 add ipython examples

HolgerKache opened the pull request:
Create root level folders (/examples and /test) and move

/test <= /python/test

/examples/python <= /python

Add new folder

/examples/ipython

Add 2 notebooks

/examples/ipython/scala_Cloudant.ipynb
/examples/ipython/python_Cloudant.ipynb

BugzId: 57606

HolgerKache committed:
Create folders for /examples and /test

Move existing python examples to /examples/python
Add ipython examples to /examples/ipython

BugzId: 57606
	deleted:    python/CloudantApp.py
	deleted:    python/CloudantDF.py
	deleted:    python/CloudantDFOption.py
	deleted:    python/test/README.md
	deleted:    python/test/conftest.py
	deleted:    python/test/helpers/utils.py
	deleted:    python/test/setup.cfg
	deleted:    python/test/test-scripts/cloudantapp/IndexOption.py
	deleted:    python/test/test-scripts/cloudantapp/RangePredicate.py
	deleted:    python/test/test-scripts/cloudantapp/RegCharPredicate.py
	deleted:    python/test/test-scripts/cloudantapp/SpecCharPredicate.py
	deleted:    python/test/test-scripts/cloudantapp/SpecCharValuePredicate.py
	deleted:    python/test/test_cloudantapp.py
	examples/
	test/

HolgerKache committed:
Add folders

	new file:   examples/ipython/python_Cloudant.html
	new file:   examples/ipython/python_Cloudant.ipynb
	new file:   examples/ipython/scala_Cloudant.html
	new file:   examples/ipython/scala_Cloudant.ipynb
	new file:   examples/python/CloudantApp.py
	new file:   examples/python/CloudantDF.py
	new file:   examples/python/CloudantDFOption.py
	new file:   test/README.md
	new file:   test/conftest.py
	new file:   test/helpers/utils.py
	new file:   test/setup.cfg
	new file:   test/test-scripts/cloudantapp/IndexOption.py
	new file:   test/test-scripts/cloudantapp/RangePredicate.py
	new file:   test/test-scripts/cloudantapp/RegCharPredicate.py
	new file:   test/test-scripts/cloudantapp/SpecCharPredicate.py
	new file:   test/test-scripts/cloudantapp/SpecCharValuePredicate.py
	new file:   test/test_cloudantapp.py

mayya-sharipova commented:
I agree with this folder structure, it is more obvious  were test and examples folders are. +1 for this.  (I have not tested two new notebooks though).

HolgerKache committed:
Updates to the Scala ipython notebook examples

Use crimes database to be consistent with Python examples

	modified:   examples/ipython/scala_Cloudant.html
	modified:   examples/ipython/scala_Cloudant.ipynb

HolgerKache committed:
Move spark-test to /examples/scala and fix build.sbt and README accordingly

	modified:   README_build.md
	modified:   README_submit.md
	deleted:    spark-test/build.sbt
	deleted:    spark-test/project/assembly.sbt
	deleted:    spark-test/project/build.properties
	deleted:    spark-test/project/eclipse.sbt
	deleted:    spark-test/src/main/scala/mytest/spark/CloudantApp.scala
	deleted:    spark-test/src/main/scala/mytest/spark/CloudantDF.scala
	deleted:    spark-test/src/main/scala/mytest/spark/CloudantDFOption.scala
	modified:   test/README.md
	examples/scala/
